.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+# Rule definitions: http://flake8.pycqa.org/en/latest/user/error-codes.html
+# W503: line break before binary operator
+exclude = venv*,__pycache__,cache
+ignore = W503
+max-complexity = 14
```

Comment:
Did you see how low you could get this number before getting errors? It’s 14 in admin and API but managed to get it down to 8 in utils: https://github.com/alphagov/notifications-utils/blob/a9394d91faefac9a5fccf9c3d49f98a681007cc4/.flake8#L6